### DISCUSSION TITLE
Please tell me your favorite sightseeing spots in Kansai area!!!

### DISCUSSION THREAD

**Discussion by tomoki0924** [NAIST-SE/AttractingContributors#140] [General]
- If you have not been to sightseeing spots in Kansai area, please type the sightseeing spots you want to go!
- By the way, Kansai is the area in Osaka, Kyoto, Hyogo and Nara (near NAIST). 
- Please fill in the list below and make pull request.

https://github.com/tomoki0924/SightseeingSpots_KANSAI

**Comment by zhangyongxin1212** [comment]
Added, plz help me too
https://github.com/zhangyongxin1212/AttractingContributors-Game

**Reply by tomoki0924** [reply]
Thankyou!
 I added!

**Reply by nagata-atsuki** [reply]
added!
please help me!!
https://github.com/nagata-atsuki/AttractingContributors-Athlete

**Comment by tomoki0924** [comment]
Please send me "pull request!"　🙇‍♂️🙇‍♂️🙇‍♂️🙇‍♂️🙇‍♂️(あと一人欲しい　at least one more, please...)
And If you want  "pull requests", please mention it to me here!
I"ll send you "pull requests"!!!!

**Reply by tomoki0924** [reply]
Thank you very much for your pull requests!! I've done to collect 10 pull requests.
If you want pull requests, please mention it the same as before.

**Comment by t-matsumoto4758** [comment]
added! please help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown
